.github/workflows/delete-stale-branches.yml

```diff
@@ -0,0 +1,21 @@
+name: Delete stale visual snapshot branches
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+permissions:
+  issues: write
```

Comment:
Why is this permission needed?

Reply:
It's supposed to create an issue before that stale branch is deleted but I suspect we don't need that for visual snapshots.

Reply:
I used a different action that is not creating issues and we would trigger manually. I don't want to run on daily schedule just for better safety.

Reply:
Nice! Thanks!